NEWS.md

```diff
@@ -1,6 +1,8 @@
 # serocalculator (development version)
 
 ## New features
+* Change default pipe setting (#312)
```

Comment:
this change still needs to go under the `## Internal changes` heading (~line 27)

Reply:
I have made the change.